Cargo.toml

```diff
@@ -1,18 +1,47 @@
 [package]
+name = "itm"
+version = "0.4.0"
+keywords = ["parse", "ARM", "ITM"]
+documentation = "https://docs.rs/itm"
 authors = [
-    "The Cortex-M Team <cortex-m@teams.rust-embedded.org>",
-    "Jorge Aparicio <jorge@japaric.io>",
+        "The Cortex-M Team <cortex-m@teams.rust-embedded.org>",
+        "Viktor Sonesten <v@tmplt.dev>",
 ]
-description = "Tool to parse and dump ITM packets"
-documentation = "https://docs.rs/itm"
 edition = "2018"
-keywords = ["parse", "ARM", "ITM"]
+readme = "README.md"
+repository = "https://github.com/embedded-rust/itm"
 license = "MIT OR Apache-2.0"
-name = "itm"
-repository = "https://github.com/japaric/itm"
-version = "0.4.0"
+description = "A decoding library and tool for the ARM Cortex-M ITM/DWT packet protocol"
+homepage = "https://github.com/embedded-rust/itm"
 
 [dependencies]
-byteorder = "1.3.0"
-thiserror = "1.0.19"
-either = "1.5.0"
+bitmatch = "0.1.1"
+bitvec = "0.22"
+thiserror = "1"
+chrono = "0.4"
+
+# only required by itm-decode executable
+anyhow = { version = "1.0", optional = true }
+structopt = { version = "0.3", optional = true }
```

Comment:
Only `itm-decode` requires these dependencies. This approach requires a user of the library to specify `default-features = false` in order to not pull these. Unsure if there is a more streamlined approach. A separate crate for `itm-decode` perhaps?

Reply:
`itm-decode` has been moved to its own crate.